engine/execution/state/delta/view.go

```diff
@@ -227,7 +234,7 @@ func (v *View) MergeView(ch state.View) error {
 
 	spockSecret := child.SpockSecret()
 
-	err := v.updateSpock(spockSecret)
+	err := v.updateSpockSecret(spockSecret)
 	if err != nil {
 		return fmt.Errorf("can not merge view: %w", err)
```

Comment:
I feel this error message could better reflect the error context:
```suggestion
		return fmt.Errorf("merging SPoCK secrets failed: %w", err)
```